### ISSUE TITLE
Detection only with a low threshold

### ISSUE THREAD

**Issue by apennisi** [AlexeyAB/darknet#1460]
@AlexeyAB 
Hi Alexey,

 I am training a classifier from scratch and also by using transfer learning but the loss starts to be stable (around 0.5) after 50000 iterations and moreover since I have 3 categories of objects, one of the categories is detected only if my threshold is 0.01. It is so weird because it is the category with the highest number of samples.

My configuration file is the standard one and I computed again the anchors.
Do I have to play with the learning rate?

Can you give me some suggestions on how to improve everything?

Thanks!

**Comment by AlexeyAB** [comment]
@apennisi Hi,

> I am training a classifier from scratch ...
> My configuration file is the standard one and I computed again the anchors.

Anchors can be applied only for Detector, not for Classifier. May be you do something wrong, and check dataset.
